apps/condo/domains/news/components/RecipientCounter.tsx

```diff
@@ -28,12 +36,13 @@ interface CounterProps {
     value: number
     type?: TypographyTitleProps['type'],
     hint?: string
+    downloader?: any
 }
 
 const styleGrayColor: CSSProperties = { color: colors.gray['5'] }
 const styleMaxWidth: CSSProperties = { maxWidth: '500px' }
 
-const Counter: React.FC<CounterProps> = ({ label, value, type = 'success', hint }) => (
+const Counter: React.FC<CounterProps> = ({ label, value, type = 'success', hint, downloader }) => (
```

Comment:
It's should be a downloadButton
```suggestion
const Counter: React.FC<CounterProps> = ({ label, value, type = 'success', hint, downloadButton }) => (
```